unitex/src/fr/umlv/unitex/config/Config.java

```diff
@@ -836,13 +852,13 @@ private static void setApplicationDir(String appPath, File s) {
 
     /**
      * Setup the UnitexToolLogger executable
-     * 
+     *
      * @param path
      *            external programs directory
      *
      * @author martinec
      */
-    public static File setupUnitexToolLogger(File path) {        
```

Comment:
Do not strip trailing spaces on the existing code as part of this PR. See http://blog.ranjansakalley.com/2015/09/cleaning-up-whitespace-changes-in-a-pull-request.html, for an approach on how-to cleaning up whitespace changes in a pull request. If you need to commit refactoring changes open a different PR prefixed with "[[refactor]](https://github.com/UnitexGramLab/unitex-doc-contributor-guidelines/blob/master/pages/05.guidelines/01.commits/docs.md)".